packages/vue_frontend_rpc/src/main.js

```diff
@@ -1,19 +1,57 @@
-import Vue from "vue";
-import App from "./App";
-
-import vuetify from "./plugins/vuetify";
+import { createApp, h } from "vue";
+import App from "./App.vue";
 import "material-design-icons-iconfont/dist/material-design-icons.css";
 import "./assets/css/globalStyles.css";
 import "@sap-devx/inquirer-gui/dist/form.css";
+import "@sap-devx/inquirer-gui-label-plugin/dist/labelPlugin.css";
```

Comment:
can you arrange the import so the css files imports would be next their relevant modules?

Reply:
done